vmware/container-service-extension
# [VCDA-1973] Client and server changes for including upgrade in apply command

Signed-off-by: Aniruddha Shamasundar <aniruddha.9794@gmail.com>

To help us process your pull request efficiently, please include: 

* Add a v36 handler for cluster apply which routes to both resize and upgrade after validation
* Deprecate `vcd cse cluster upgrade` command for api version 36.0

Testing done:
* create a cluster
* upgrade and resize the cluster at the same time and check if it throws an error
* upgrade a cluster
* resize a cluster

@rocknes @sakthisunda @sahithi @ChintanShahVMware @arunmk @ltimothy7


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/container-service-extension/853)
